### ISSUE TITLE
src/lib/libm/noieee_src/n_fmod.c: 4 * bad printf ?

### ISSUE THREAD

**Issue by dcb314** [Stichting-MINIX-Research-Foundation/minix#102]
[src/lib/libm/noieee_src/n_fmod.c:125]: (error) printf format string requires 3 parameters but only 2 are given.

       (void)printf(" x    = 0x%08.8x %08.8x (%24.16e)\n",x,x);
        (void)printf(" y    = 0x%08.8x %08.8x (%24.16e)\n",y,y);
        (void)printf(" fmod = 0x%08.8x %08.8x (%24.16e)\n",ro,ro);
        (void)printf("_fmod = 0x%08.8x %08.8x (%24.16e)\n",rn,rn);

**Comment by dcvmoole** [comment]
Thank you for your bug report. However, this issue is in NetBSD code that we simply import as is. In fact, basically anything outside the minix/ subdirectory of the MINIX3 source tree is imported from NetBSD. Except in particularly serious cases, we try to avoid changing such imported code so as to limit the amount of work necessary to synchronize with newer NetBSD code later. Instead, we highly recommend that you file this bug report with NetBSD, so that they can fix their code; at a later stage we will then end up importing the fix again.

In other words, we would love to hear about any issues in the minix/ part of the source tree, but for almost everything else, the NetBSD people will want to hear about it instead. In any case, your efforts are appreciated.
